src/utils/api/callout.ts

```diff
@@ -108,13 +108,18 @@ export async function fetchResponsesForMap(
 
 export async function createResponse(
   slug: string,
-  data: CreateCalloutResponseData
+  data: CreateCalloutResponseData,
+  captchaToken?: string
 ): Promise<void> {
-  await instance.post(`/callout/${slug}/responses`, {
-    answers: data.answers,
-    guestName: data.guestName,
-    guestEmail: data.guestEmail,
-  });
+  await instance.post(
+    `/callout/${slug}/responses`,
+    {
+      answers: data.answers,
+      guestName: data.guestName,
+      guestEmail: data.guestEmail,
+    },
+    { params: { captchaToken } }
```

Comment:
It is no problem to send the parameter even if it is not required?

Reply:
If `captchaToken` is undefined it won't actually send the parameter I think, although I haven't actually checked that